joomla/joomla-cms
# Allow plugin overrides via the backend template manager

### Summary of Changes

Allow plugin overrides via the backend template manager

### Testing Instructions

- Apply this patch
- go to Extensions -> Templates -> Templates -> Protostar Details and Files
- Switch to the `Create Overrides` tab
- confirm there are also the plugins that you can override in that list.
- hit the override button for `plg_twofactorauth_totp`
- confirm it creates and executes the new override
- change the view using the new override
- confirm the new override is used.

### Expected result

Plugins that supports overrides are in the list to create a override using the GUI

![image](https://cloud.githubusercontent.com/assets/2596554/23827706/d10f79ac-06b9-11e7-88a2-360ed43ba1c6.png)


### Actual result

No Plugin are in the list in the backend.

![image](https://cloud.githubusercontent.com/assets/2596554/23827715/fff9c22c-06b9-11e7-85ac-dbb1704a7b7e.png)

### Documentation Changes Required

Nothing needs to be changed as there is no doku about that feature on the help screens. Maybe we should create something like this but this is a different issue and out of scope of that PR.
